packages/assets-controllers/src/TokenRatesController.ts

```diff
@@ -527,15 +570,38 @@ export class TokenRatesController extends StaticIntervalPollingControllerV1<
       },
       initialResult: {},
     });
+    contractNativeInformations = tokenPricesByTokenAddress;
 
-    return Object.entries(tokenPricesByTokenAddress).reduce(
+    // fetch for native token
+    if (tokenAddresses.length === 0) {
+      const ZERO_ADDRESS = '0x0000000000000000000000000000000000000000';
```

Comment:
Can you explain why this is a separate case?  Why can't the caller explicitly pass the zero address if they want native data?

Reply:
We'll use the zero address to access market data for the native token. This means that, regardless of whether users have imported any tokens or not, we're required to initiate at least one request using the zero address to gather market information about the native token.

Reply:
Can we explain that somewhere here, perhaps in a comment? I also did not understand why that was necessary just by looking at this.

Besides this, it seems that we are already passing the zero address to the API directly in `fetchTokenPrices`. So, do we have to do this here?

Reply:
@mcmire is this means that we'll make two calls api ? one for native chips and the other for erc20s ?